src/python/WMCore/ReqMgr/Web/utils.py

```diff
@@ -264,3 +271,18 @@ def sort(docs, sortby):
     "Sort given documents by sortby attribute"
     for doc in docs:
         yield doc
+        
+def reorder_list(org_list, first):
+    """
+    if the first is in the list.
+    move the first in front of the list
+    if not, add first to the list
+    """
+    new_list = list(org_list)
+    try:
+        i = new_list.index(first)
+        new_list[0], new_list[i] = new_list[i],  new_list[0]
```

Comment:
BTW, I'm not sure the case `if not, add first to the list` can ever happen. And if it does, that would be for a good reason (like cmssw that got deprecated, and in this case I'd be in favor of not adding new values on top of the original source of information)